### PR TITLE
Forms Automated Test - LPS-131234 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
@natocesarrego , @diogocordeiro these should be ready for review!

**Comment by diogocordeiro** [comment]
ci:test:forms-functional

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path [hunk 1]
```diff
@@ -95,6 +95,11 @@
 	<td></td>
 </tr>
 <!--TOOLBAR-->
+<tr>
```
Comment (diogocordeiro) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path:98]: Hi @leticiachiu ,

Although the CKEditor.path file exists this test is in the context of forms, we prefer that Paths be kept in our own files. Could you please remove this path from this file and add it on the FormFields.path file instead.

Thanks!

**Review by diogocordeiro — CHANGES_REQUESTED** [review]

**Comment by diogocordeiro** [comment]
ci:forward

**Pull request closed without merging**